impl/src/main/java/org/jboss/cdi/tck/tests/lookup/injectionpoint/TransientFieldInjectionPointBean.java

```diff
@@ -29,7 +29,7 @@
 @RequestScoped
 public class TransientFieldInjectionPointBean {
     @Inject
-    @AnimalStereotype
+    @Animal
```

Comment:
Is there a point in having any annotation here? The related test only checks the metadata to see if the IP is transient.

Reply:
Not really, that annotation isn't used anywhere. I can remove it too.

Reply:
Done.